src/ensembl/.eslintrc.js

```diff
@@ -3,9 +3,10 @@ module.exports = {
   extends: [
     'plugin:@typescript-eslint/recommended',
     'plugin:react/recommended',
-    'plugin:prettier/recommended' // Displays prettier errors as ESLint errors. This has to be the last in this array (not yet sure why).
+    'plugin:prettier/recommended', // Displays prettier errors as ESLint errors. This has to be the last in this array (not yet sure why).
+    'plugin:jest/style'
   ],
-  plugins: ['react-hooks'],
+  plugins: ['react-hooks', 'eslint'],
```

Comment:
That's different from `plugins: ['jest']` as described in the [docs](https://github.com/jest-community/eslint-plugin-jest#usage). What does the eslint plugin do, and why did you prefer `'plugin:jest/style'` and `plugins: ['eslint']` to `plugins: ['jest']`?

Reply:
`plugins: ['eslint`] is a mistake. I'll fix that. `plugin:jest/style` is to add more stylistic rules for jest. I thought of including that and see if it can spot more issues for us.

Reply:
✅